superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.jsx

```diff
@@ -161,79 +98,73 @@ export default class SqlEditorLeftBar extends React.PureComponent {
     </IconTooltip>
   );
 
-  render() {
-    const shouldShowReset = window.location.search === '?reset=1';
-    const tableMetaDataHeight = this.props.height - 130; // 130 is the height of the selects above
-    const qe = this.props.queryEditor;
-    return (
-      <div className="SqlEditorLeftBar">
-        <TableSelector
-          database={this.props.database}
-          dbId={qe.dbId}
-          getDbList={this.getDbList}
-          handleError={this.props.actions.addDangerToast}
-          onDbChange={this.onDbChange}
-          onSchemaChange={this.onSchemaChange}
-          onSchemasLoad={this.onSchemasLoad}
-          onTableChange={this.onTableChange}
-          onTablesLoad={this.onTablesLoad}
-          schema={qe.schema}
-          sqlLabMode
-        />
-        <div className="divider" />
-        <StyledScrollbarContainer>
-          <StyledScrollbarContent contentHeight={tableMetaDataHeight}>
-            <Collapse
-              activeKey={this.props.tables
-                .filter(({ expanded }) => expanded)
-                .map(({ id }) => id)}
-              css={theme => css`
-                .ant-collapse-item {
-                  margin-bottom: ${theme.gridUnit * 3}px;
-                }
-                .ant-collapse-header {
-                  padding: 0px !important;
-                  display: flex;
-                  align-items: center;
-                }
-                .ant-collapse-content-box {
-                  padding: 0px ${theme.gridUnit * 4}px 0px 0px !important;
-                }
-                .ant-collapse-arrow {
-                  top: ${theme.gridUnit * 2}px !important;
-                  color: ${theme.colors.primary.dark1} !important;
-                  &: hover {
-                    color: ${theme.colors.primary.dark2} !important;
-                  }
+  const shouldShowReset = window.location.search === '?reset=1';
+  const tableMetaDataHeight = height - 130; // 130 is the height of the selects above
+
+  return (
+    <div className="SqlEditorLeftBar">
+      <TableSelector
+        database={database}
+        dbId={queryEditor.dbId}
+        getDbList={actions.setDatabases}
+        handleError={actions.addDangerToast}
+        onDbChange={onDbChange}
+        onSchemaChange={actions.queryEditorSetSchema}
+        onSchemasLoad={actions.queryEditorSetSchemaOptions}
+        onTableChange={onTableChange}
+        onTablesLoad={actions.queryEditorSetTableOptions}
+        schema={queryEditor.schema}
+        sqlLabMode
+      />
+      <div className="divider" />
+      <StyledScrollbarContainer>
+        <StyledScrollbarContent contentHeight={tableMetaDataHeight}>
```

Comment:
```suggestion
        <div
          css={css`
            height: ${props => props.contentHeight}px;
          `}
          contentHeight={tableMetaDataHeight}
        >
```
Since this style definition is only one line, it doesn't need to be in an external variable.